lib/maropost/contact.rb

```diff
@@ -53,6 +53,18 @@ def list_parameters
       { cell_phone_number: cell_phone_number }.merge(lists)
     end
 
+    def merge_settings(old_contact)
```

Comment:
I dont think you need this if you leverage the object attributes

Reply:
It is not a simple merge.